packages/wrangler/src/__tests__/versions/secrets/put.test.ts

```diff
@@ -380,7 +380,7 @@ describe("versions secret put", () => {
 						),
 						"module.wasm"
 					);
-
+					// @ts-expect-error this is fine
```

Comment:
This an MSW types incompatibility, but I'm pretty sure it should work fine at runtime in tests. I imagine it'll likely be solved by upgrading the MSW version, but I wanted to reduce the complexities in this PR.